Trappist/src/Promact.Trappist.Web/wwwroot/app/conduct/conduct.service.ts

```diff
@@ -0,0 +1,19 @@
+﻿import { Injectable } from '@angular/core';
+import { HttpService } from '../core/http.service';
+
+@Injectable()
+export class ConductService {
+    private registerTestAttendeeApiUrl: string = '/api/conduct/register';
```

Comment:
The path should be "conduct/:testid/register". make appropriate changes wherever required.